src/tigerbeetle/benchmark_load.zig

```diff
@@ -126,7 +126,11 @@ pub fn main(
     }
 
     // If no seed was given, use a default seed for reproducibility.
-    const seed: usize = cli_args.seed orelse 42;
+    const seed = seed_from_arg: {
+        const seed_argument = cli_args.seed orelse break :seed_from_arg 42;
+        break :seed_from_arg vsr.testing.parse_seed(seed_argument);
```

Comment:
Hm, using vsr.testing namespace here feels wrong, as this is code from the main binary, so it _shouldn't_ include that namespace. Sort-of the same reason why `benchamrk_driver` doesn't use `shell.zig` and spanwns the process by hand. 

But we already use a couple of other functions here, so this is per-existing and doesn't need fixing in this PR